src/components/Wallet/constants.js

```diff
@@ -1,3 +1,3 @@
 export const CARAVAN_CONFIG = "caravan_config";
-
-export default CARAVAN_CONFIG;
```

Comment:
We'll still want a default export here in order to satiate eslint, maybe just an object containing the constants?

Reply:
Lint isn't throwing any errors for me on this. It did when there was only one export but I think it's ok if there are multiple.

Reply:
Feel free to resolve this conversation.